src/renderer/mod.rs

```diff
@@ -62,15 +62,28 @@ impl Renderer {
 
     /// Default terminal styling
     pub const fn styled() -> Self {
+        const BRIGHT_BLUE: Style = if cfg!(windows) {
+            AnsiColor::BrightCyan.on_default().effects(Effects::BOLD)
+        } else {
+            AnsiColor::BrightBlue.on_default().effects(Effects::BOLD)
+        };
         Self {
             stylesheet: Stylesheet {
                 error: AnsiColor::BrightRed.on_default().effects(Effects::BOLD),
-                warning: AnsiColor::BrightYellow.on_default().effects(Effects::BOLD),
-                info: AnsiColor::BrightBlue.on_default().effects(Effects::BOLD),
-                note: Style::new().effects(Effects::BOLD),
+                warning: if cfg!(windows) {
+                    AnsiColor::BrightYellow.on_default().effects(Effects::BOLD)
+                } else {
+                    AnsiColor::Yellow.on_default().effects(Effects::BOLD)
+                },
+                info: BRIGHT_BLUE,
+                note: AnsiColor::BrightGreen.on_default().effects(Effects::BOLD),
                 help: AnsiColor::BrightCyan.on_default().effects(Effects::BOLD),
-                line_no: AnsiColor::BrightBlue.on_default().effects(Effects::BOLD),
-                emphasis: Style::new().effects(Effects::BOLD),
+                line_no: BRIGHT_BLUE,
+                emphasis: if cfg!(windows) {
```

Comment:
If I'm reading this correctly, this maps to `MainHeaderMsg`?  What about `Highlight`?

Reply:
Correct, `emphasis` maps to `MainHeadingMsg`. As for `Highlight` `annotate-snippets` doesn't currently have anything for that concept (that I can tell).